packages/grafana-ui/src/components/Table/Table.tsx

```diff
@@ -27,7 +27,6 @@ import { Icon } from '../Icon/Icon';
 import { CustomScrollbar } from '../CustomScrollbar/CustomScrollbar';
 import { Filter } from './Filter';
 import { TableCell } from './TableCell';
-import ClipboardButtonStoryInternal from '../ClipboardButton/ClipboardButton.story.internal';
```

Comment:
Not sure how this got inserted here but should be safe to remove right @torkelo?